CHANGELOG.md

```diff
@@ -13,8 +13,10 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-14336: product `BuildDate` attribute was removed. It won't be returned by
 `/debug/versions.json` endpoint and `roxctl version --json` command.
 
-### Deprecated Fatures
+### Deprecated Features
 - Deprecated `/v1/telemetry/configure` service.
+- The `expiration` field in the `Exclusion` proto will be deprecated in future releases. 
```

Comment:
```suggestion
- The `expiration` field in the `Exclusion` proto is deprecated and will be removed in the 4.2 release.
```

Reply:
Are we saying out release numbers? If you see precedents we are not.

Reply:
> Label and Annotation search options are deprecated and will be removed in 3.73
https://github.com/stackrox/stackrox/blob/813cee1c284da8b274b405c3fdb4d0649c040317/CHANGELOG.md?plain=1#L144

https://github.com/stackrox/stackrox/blob/813cee1c284da8b274b405c3fdb4d0649c040317/CHANGELOG.md?plain=1#L176

There are a few examples. Not a big deal though since it gets converted into a table in the real docs